src/Utilities/ProjectData.h

```diff
@@ -134,6 +142,9 @@ class ProjectData
 
   // The driver version of the project
   DriverVersion driver_version_;
+
+  // tracks if complex wavefunction is used, true: complex, false: real
+  bool is_complex_;
```

Comment:
This needs to be initialized. Probably for now based on QMC_COMPLEX

Reply:
Please do a rebase once this is fixed.

Reply:
Thanks @ye-luo , that was a complete oversight from my side. Fixed it.

Reply:
Our CI actually reported failure. So it well served as a test for CI.

Reply:
Interesting, I am using vanilla Ubuntu 22 and the local unit tests passed. The gcc compiler might be too permissive.